configure.ac

```diff
@@ -763,8 +763,23 @@ AC_LINK_IFELSE([AC_LANG_SOURCE([
   }
   ])],
   [
-    AC_DEFINE(HAVE_THREAD_LOCAL,1,[Define if thread_local is supported.])
-    AC_MSG_RESULT(yes)
+   case $host in
+     *mingw*)
+        # mingw32's implementation of thread_local has also been shown to behave
+        # erroneously under concurrent usage; see:
+        # https://gist.github.com/jamesob/fe9a872051a88b2025b1aa37bfa98605
+        AC_MSG_RESULT(no)
+        ;;
+      *darwin*)
+        # TODO enable therad_local on later versions of Darwin where it is
```

Comment:
In commit "disable thread_local on unreliable platforms" (696bdb751055c01c645dc95df486d56a873325de)

Little spelling mistake

Reply:
D'oh. Fixed.